dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/deduplication/hibernate/HibernatePotentialDuplicateStore.java

```diff
@@ -231,19 +231,9 @@ public void removeTrackedEntity( TrackedEntityInstance trackedEntityInstance )
     @Override
     public void auditMerge( DeduplicationMergeParams params )
     {
```

Comment:
To me, this change is out of the scope of this PR. Also why are we removing it?

Reply:
Remember we are merging into a custom branch, not a version branch, so I think its ok if we mix it a bit. The reason I just added it here, is since it also related to the persistence, even though its auditing.

Except for relationships, everything is merged automatically through the getSession.save()/update(). So I removed it to avoid duplicate audits.